CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Do we want to add in things about content submitted here should only be relevant to scientific workflows or something?

Reply:
We should probably add some language about this, I think.